### ISSUE TITLE
Testing: Change name of `NO_XDIST` flag

### ISSUE THREAD

**Issue by rdimaio** [rucio/rucio#7010]
https://github.com/rucio/rucio/blob/8c4672c64b8784fd9f9cdc9e3516d7d83ecd22c9/tools/pytest.sh#L33-L58

`NO_XDIST=False` means that XDIST is true, so for example:

```
if [[ "$NO_XDIST" == "False" ]]; then
```

means "let's run `xdist`"

I think it would be easier to comprehend if it was named `RUN_XDIST`, `IS_XDIST` or something like that - the `NO_` part causes it to be a bit confusing due to the double negation

**Comment by mgajek-cern** [comment]
@rdimaio Is this GH issue still valid ? If so I will add it to the current sprint and will address it in an initial PR to get familiar with the process

**Comment by rdimaio** [comment]
> [@rdimaio](https://github.com/rdimaio) Is this GH issue still valid ? If so I will add it to the current sprint and will address it in an initial PR to get familiar with the process

Yeah, I think it's still valid, a better naming convention would be useful here in my opinion
